package.json

```diff
@@ -51,9 +51,8 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "lerna run lint:scss",
-    "prepush": "node scripts/prepush/index.js",
     "pretest": "npm run lint",
-    "postinstall": "npm run bootstrap",
+    "postinstall": "npm run compile && npm run bootstrap",
```

Comment:
https://travis-ci.org/cerner/terra-core/builds/237962719#L426

It is being executed, but it collapses in the log.